controllers/extension-certificate-service/charts/cert-broker-rbac/templates/rbac.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: garden.sapcloud.io:system:cert-broker
```

Comment:
Hm, I know that I previously suggested to rename it back to `garden.sapcloud.io`, but thinking again about it: When this extension is introduced in Gardener then this old role will be deleted anyways, because it's currently managed by the kube-addon-manager. So, we can also choose a new name here, right? WDYT about `extensions.gardener.cloud:extension-certificate-service:cert-broker`? (E.g., the AWS provider will create a ClusterRole for MCM named `extensions.gardener.cloud:provider-aws:machine-controller-manager`).
(Same for all other `garden.sapcloud.io` names.)

Reply:
Sounds good!